modules/021-cni-cilium/images/base-cilium-dev/werf.inc.yaml

```diff
@@ -0,0 +1,112 @@
+{{- $goVersion := "1.21.5" }}
+{{- $protocVersion := "22.3" }}
+{{- $bazelVersions := "3.7.0 3.7.1 3.7.2 6.1.0" }}
+{{- $fromCacheVersion := "16" }}
+---
+# #####################################################################
+# BASE_DEV image for build binaries of all cilium components (based on Ubuntu)
+# based on original images: runtime, builder, compilers, cilium-envoy, iptables
+# Based on https://github.com/cilium/cilium/blob/v1.14.5/images/runtime/Dockerfile
+# and https://github.com/cilium/cilium/blob/v1.14.5/images/runtime/install-runtime-deps.sh
+# and https://github.com/cilium/cilium/blob/v1.14.5/images/builder/Dockerfile
+# and https://github.com/cilium/image-tools/blob/063fa0d879d9560b85b6134c48ece08e672fa057/images/compilers/Dockerfile
+# and https://github.com/cilium/image-tools/blob/063fa0d879d9560b85b6134c48ece08e672fa057/images/compilers/install-deps.sh
+# and https://github.com/cilium/image-tools/blob/063fa0d879d9560b85b6134c48ece08e672fa057/images/compilers/install-bazel.sh
+# and https://github.com/cilium/cilium/blob/v1.14.5/images/cilium/Dockerfile#L9
+# and https://github.com/cilium/proxy/blob/ad82c7c56e88989992fd25d8d67747de865c823b/Dockerfile.builder
+# and https://github.com/cilium/image-tools/blob/3e6b0f588d788ac0e69f4d126cfb3c55b1513948/images/iptables/Dockerfile
+#
+# Original IMAGE_TAG of compilers (taken from builds of BPFTOOL and LLVM) is 5569a29cea6b3ad50aeb03102aaf3dc03841197c
+# Corresponding commit is 063fa0d879d9560b85b6134c48ece08e672fa057 (https://github.com/cilium/image-tools/tree/063fa0d879d9560b85b6134c48ece08e672fa057)
+# Original IMAGE_TAG of cilium-envoy is v1.26.6-ad82c7c56e88989992fd25d8d67747de865c823b
+# Corresponding commit is ad82c7c56e88989992fd25d8d67747de865c823b
+# Original IMAGE_TAG of cilium/iptables is 67f517af50e18f64cd12625021f1c39246bb4f92
+# Corresponding commit is 3e6b0f588d788ac0e69f4d126cfb3c55b1513948 (https://github.com/cilium/image-tools/tree/3e6b0f588d788ac0e69f4d126cfb3c55b1513948)
+# #####################################################################
+---
+image: {{ $.ModuleName }}/base-cilium-dev
+from: {{ $.Images.BASE_UBUNTU }}
+fromCacheVersion: {{ $fromCacheVersion }}
+mount:
+- fromPath: ~/go-pkg-cache
+  to: /go/pkg
```

Comment:
remove pls :(

Reply:
It has been decided to leave this for now.